cms/templates/widgets/header.html

```diff
@@ -36,12 +35,10 @@ <h1 class="branding"><a href="/">
             grading_url = reverse('contentstore.views.grading_handler', kwargs={'course_key_string': unicode(course_key)})
             advanced_settings_url = reverse('contentstore.views.advanced_settings_handler', kwargs={'course_key_string': unicode(course_key)})
             tabs_url = reverse('contentstore.views.tabs_handler', kwargs={'course_key_string': unicode(course_key)})
+            certificates_url = ''
```

Comment:
ahhh, I see it now, yes -- sorry, just getting back into the swing of things here :)  okay, :+1: for me, then